Makefile

```
@@ -0,0 +1,21 @@
+PYTHON ?= python3
+
+.PHONY: install dev-install test lint format start-node
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)